tests/robot/CONDITION/01_create.robot

```diff
@@ -36,7 +36,7 @@ Force Tags              create
     ...                 2. trigger condition endpoint
 
     ehr.create new ehr    000_ehr_status.json
-    condition.create diagnose condition    create.json
+    condition.create diagnose condition    create-default.json
```

Comment:
I will refactor this in your other branch 
https://github.com/ehrbase/fhir-bridge-poc/tree/feature/NUM-544_Repro_refactoring-Condition